src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -1,6 +1,9 @@
 package lotto.domain
 
-class Lotto(val numbers: Set<Int>) {
+class Lotto(val numbers: Set<Int>, val type: LottoType) {
+
+    constructor(numbers: Set<Int>) : this(numbers.toSet(), LottoType.AUTO)
```

Comment:
아래와 같이 기본 값을 주는 형태로도 사용할 수 있지 않을까요?

```suggestion
class Lotto(val numbers: Set<Int>, val type: LottoType = LottoType.AUTO) {
```